RedHatInsights/vulnerability-ui
# Table items not selectable when no action is available

https://issues.redhat.com/browse/VULN-2501

## To test:
- Load up the app as a user with Vulnerability admin role -- the app should work as regular -- checkboxes are available on all tables with corresponding actions per each table:
  - CVE list -- update status, update business risk
  - CVE detail and System detail -- update status
  - System list -- enable/disable system
- Load up the app as a user with only Vulnerability viewer role -- no table should now show checkboxes next to the items and corresponding action buttons should not be available in table row kebab nor toolbar kebab
